deltachat-ios/DC/Wrapper.swift

```diff
@@ -153,6 +153,10 @@ class DcContext {
        }
        return nil
     }
+
+    func saveChatAvatarImage(chatId: Int, path: String) {
```

Comment:
in general, i'd give these one-to-one wrappers for the core functions the same name as the core function. the case can be adapted. so eg. setChatProfileImage() in this case.

however, this should not block merging.